src/Estimators/SpinDensityNew.cpp

```diff
@@ -48,14 +52,9 @@ SpinDensityNew::SpinDensityNew(SpinDensityInput&& input,
       lattice_(lattice)
 {
   my_name_ = "SpinDensity";
-  std::cout << "SpinDensity constructor called\n";
   data_locality_ = dl;
   if (input_.get_cell().explicitly_defined == true)
-    throw std::runtime_error(
-        "SpinDensityNew should not be constructed with both a cell in its input and an lattice input argument.");
-  else if (lattice_.explicitly_defined == false)
-    throw std::runtime_error("SpinDensityNew cannot be constructed from a lattice that is not explicitly defined");
-
+    lattice_ = input_.get_cell();
```

Comment:
changes in this file don't seem belonging to this topic. Confused.

Reply:
These were to made handling of input in for SpinDensityNew out of EstimatorManagerNew and into SpinDensityNew itself, see EstimatorManagerNew::put. The new EstimatorManagerNew constructor doesn't look inside of estimator input objects.